server/src/main/java/com/microsoft/java/bs/core/internal/gradle/GradleApiConnector.java

```diff
@@ -93,10 +96,10 @@ public GradleSourceSets getGradleSourceSets(URI projectUri) {
       // since the model returned from Gradle TAPI is a wrapped object, here we re-construct it
       // via a copy constructor and return as a POJO.
       return new DefaultGradleSourceSets(customModelBuilder.get());
-    } catch (GradleConnectionException | IllegalStateException e) {
-      summary = e.getMessage();
+    } catch (GradleConnectionException | IllegalStateException | IOException e) {
+      summary = e.getMessage() + "\n" + errorOut.toString();
```

Comment:
Let's check if errorOut is empty or not before appending to the summary text.

Reply:
Done